src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -308,7 +295,8 @@ PolicyHandler::~PolicyHandler() {}
 
 PolicyEncryptionFlagGetterInterfaceSPtr
 PolicyHandler::PolicyEncryptionFlagGetter() const {
-  return policy_manager_;
+  DCHECK_OR_RETURN(atomic_policy_manager_, nullptr);
+  return atomic_policy_manager_;
```

Comment:
@IGapchuk I can see the duplicate of atomic_policy_manager_ getter here:https://github.com/smartdevicelink/sdl_core/pull/3497/files#diff-7ec0ea8e1e752c736fb05b5f89890fbfR774 (function LoadPolicyManager) 
Please leave only one getter

Reply:
@LitvinenkoIra The `LoadPolicyManager` method in the base changes in this PR and the `PolicyEncryptionFlagGetter` method is the interface method. So we decided to combine the approaches and call `LoadPolicyManager` inside the `PolicyEncryptionFlagGetter`.